builders/scripts/builder_darwin_amd64_wine

```diff
@@ -4,7 +4,7 @@ cp -a "/root/wine-git" "/root/wine-tools" || exit 1
 ####### Build Tools
 echo "[STAGE 1/11] Configure tools"
 cd "/root/wine-tools"
-./configure || exit 2
+./configure --enable-win64 || exit 2
```

Comment:
Are you sure that you need enable-win64 with wine-tools?

Reply:
Yes since were now using amd64 image, doing it wont affect the compiled wine.
Winehq actually do this with there builder

Reply:
Ok